src/Cardano/Shell/Presets.hs

```diff
@@ -0,0 +1,316 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module Cardano.Shell.Presets
+    ( mainnetConfiguration
+    , devConfiguration
+    ) where
+
+import           Cardano.Prelude
+
+import           Cardano.Shell.Types (ApplicationEnvironment (..), Block (..),
+                                      BlockVersionData (..),
+                                      CardanoApplication (..),
+                                      CardanoConfiguration (..),
+                                      CardanoEnvironment (..),
+                                      CardanoFeature (..),
+                                      CardanoFeatureInit (..), Certificate (..),
+                                      Core (..), DLG (..), FakeAvvmBalance (..),
+                                      Genesis (..), Initializer (..),
+                                      LastKnownBlockVersion (..), NTP (..),
+                                      NoDependency (..), Node (..),
+                                      ProtocolConstants (..), SSC (..),
+                                      SoftForkRule (..), Spec (..), TLS (..),
+                                      TXP (..), TestBalance (..), Throttle (..),
+                                      TxFeePolicy (..), TxSizeLinear (..),
+                                      Update (..), Wallet (..),
+                                      applicationProductionMode,
+                                      initializeCardanoEnvironment,
+                                      loadCardanoConfiguration)
+
+--------------------------------------------------------------------------------
+-- Cardano Mainnet Configuration
+--------------------------------------------------------------------------------
+
+mainnetConfiguration :: CardanoConfiguration
+mainnetConfiguration = CardanoConfiguration
+                       { ccLogPath  = "./logs/"
+                       , ccLogConfig = "./log-config.yaml"
+                       , ccDBPath   = "./db/"
+                       , ccApplicationLockFile = ""
+                       , ccCore       = Core { coGenesis =
```

Comment:
Can you format this better? It's still all over the place.

Reply:
Not sure how to properly style this. Stylish haskell isn't fixing it for me automatically, and I haven't had nested records this deep before. Any recommendations?

Reply:
Manual formatting is what I tend to do?

Reply:
I tried another formatting structure. Any recommended changes @ksaric ?